Ex10_2.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import scipy
+import scipy.integrate as spint
+from plotnine import *
+
+
+def diseaseSim(y, t0, B, gam):
+    S=y[0]
+    I=y[0]
+    R=y[0]
+    
+    dSdt=-B*I*S
+    dIdt=(B*I*S)-(gam*I)
+    dRdt=gam*I
+    
+    return(dSdt, dIdt, dRdt)
+    
+beta=[0.0005,0.005,0.0001,0.00005,0.0001,0.0002,0.0001]
+gamma=[0.05,0.5,0.1,0.1,0.05,0.05,0.06]
+
+times=range(0,500)
+#case1
+
+
+#case 1
+y0=[999,1,0]
+params=(beta[0],gamma[0])
+sim=spint.odeint(func=diseaseSim,y0=y0,t=times,args=params)
+simDF_1=pd.DataFrame({"t":times,"S":sim[:,0],"I":sim[:,1],"R":sim[:,2]})
+plot_1=ggplot(simDF_1,aes(x="t", y="S"))+geom_line(color='blue')+geom_line(simDF_1,aes(x="t",y="I"), color='red')+ geom_line(simDF_1,aes(x="t",y="R"), color='green')+theme_classic()
+plot_1
+#maximum daily incidence is 292.26, occuring between day 0 and 1
+max(simDF_1.iloc[:,0])
```

Comment:
The simulation is good. The max daily incidence should be:
numpy.max(sim[1:len(sim),1]-sim[0:(len(sim)-1),1])